File: app/src/main/java/com/example/donald/kotlinjacocoexample/TextBuilder.java
package com.example.donald.kotlinjacocoexample;

public class TextBuilder {
    public String getReturnString(String button) {
        if(button == "one") {
            return "You returned 1";
        }
        if(button == "two") {
            return "You returned 2";
        }
        return "OOPS";
    }
}

File: app/src/main/java/com/example/donald/kotlinjacocoexample/TextBuilder.kt
package com.example.donald.kotlinjacocoexample

class TextBuilder {
    fun getReturnString(button: String): String {
        if (button === "one") {
            return "You returned 1"
        }
        return if (button === "two") {
            "You returned 2"
        } else "OOPS"
    }
}